versions/3.0.md

```diff
@@ -1,5 +1,133 @@
 # OpenAPI Specification
 
+### DO NOT MERGE
+
+These are comments for discussion and should be turned into separate PRs
+
+#### NOTES
+
+## Both HTTP and HTTPS ports cannot be specified #214
+
+Proposal: We have schemes, host (+ port), basePath, all which are typically linked to gether.  Suggest we produce a new object:
+
+```yaml
+hosts:
+  - scheme: http
+    host: foo.com
+    port: 8080
+    basePath: /bar
+  - scheme: https
+    host: foo.com
+    port: 8443
+    basePath: /baz
+```
+
+And remove the separate `host`, `basePath`, `schemes`.  Because the port will never be the same between http, https, it makes no sense to support multiple ones
+
+Fixes #489, #214
+
+
+## Why don't host and basePath support path templating? #169
+
+- [ ] Proposal: this belongs under parent #574
+
+## Support an operation to have multiple specifications per path (e.g. multiple POST operation per path) #182
+
+- [ ] Proposal: this is a payload issue, move to parent #586
+
+## Allow for overloading/aliasing of route path #213
+
+- [ ] Proposal: change in #70 would address this
+
+## Allow references between operations inside descriptions. #239
+
+- [ ] Proposal: this is really about _links_ and belongs in the #586 issue
+
+## Swagger.next: Let's not use hashes anymore #208
+
+- [ ] Proposal: while true, moving to arrays causes more problems than it solves.  Suggest we close this with no change.
+
+## Group multiple parameter definitions for better maintainability #445
+
+- [ ] Proposal: close.  Using global parameters will help with this, and references to arrays of parameters is just wrong.a
+
+
+## Allow documenting HATEOAS APIs (pathless operations) #577
+
+- [ ] Proposal: do not support pathless operations.  It's against the spirit of the specification as being declarative.  The support for links _inside_ a path though, will be covered by #574 and #586.  Remove reference to #560 and link to others.
```

Comment:
If the server is self-documenting and has no predefined paths, you don't need OAS IMHO.  "/" is a root path.